starter-code/css/main.css

```diff
@@ -1,10 +1,87 @@
-/*
+    /*
 Colors to use:
 dark blue: #06365f
 light blue: #1c5380
 */
 
 body {
-  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
-  margin: 0;
+    font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
+    margin: 0;
 }
+
+#top-section {
+    width: 100%;
+    height: 250px;
+    background-color: #1c5380;
+}
+
+.phones {
+    float: left;
+    margin-top: 100px;
+    margin-left: 60px;
+}
+
+.brand {
+    float: left;
+    position: relative;
+    margin-top: 110px;
+    margin-left: 55px;
+}
+
+.log-in {
```

Comment:
Changing this from a <div> to a <button> would make the element blend in with the background color less.